omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -148,29 +152,29 @@ def panel_to_page_coords(self, shape_x, shape_y):
         the cropped panel region
         """
         rotation = self.panel['rotation']
-        # img coords: centre of rotation
-        cx = self.crop['x'] + (self.crop['width']/2)
-        cy = self.crop['y'] + (self.crop['height']/2)
-        dx = cx - shape_x
-        dy = cy - shape_y
-        # distance of point from centre of rotation
-        h = sqrt(dx * dx + dy * dy)
-        # and the angle (avoid division by zero!)
-        if dy == 0:
-            angle1 = 90
-        else:
+        if rotation != 0:
+            # img coords: centre of rotation
+            cx = self.crop['x'] + (self.crop['width']/2)
+            cy = self.crop['y'] + (self.crop['height']/2)
+            dx = cx - shape_x
+            dy = cy - shape_y
+            # distance of point from centre of rotation
+            h = sqrt(dx * dx + dy * dy)
+            # and the angle (avoid division by zero!)
+            if dy == 0:
+                dy = 0.000001
```

Comment:
what is the rationale?

Reply:
You can't divide by zero, but dividing by a number close to zero works very well and is the simplest workaround (the previous workaround was not working).

Reply:
Isn't this the case we you want to use `math.atan2` rather than `math.atan`

Reply:
Yep, that works. Thanks